stl/inc/memory

```diff
@@ -284,34 +284,139 @@ _NoThrowFwdIt uninitialized_fill_n(_NoThrowFwdIt _First, const _Diff _Count_raw,
 }
 #endif // _HAS_IF_CONSTEXPR
 
+// FUNCTION TEMPLATE construct_at
 #if _HAS_CXX20
 template <class _Ty, class... _Types>
-auto construct_at(_Ty* const _Location, _Types&&... _Args) noexcept(noexcept(::new (const_cast<void*>(
-    static_cast<const volatile void*>(_Location))) _Ty(_STD forward<_Types>(_Args)...))) // strengthened
+_CONSTEXPR20_DYNALLOC auto construct_at(_Ty* const _Location, _Types&&... _Args) noexcept(
+    noexcept(::new (const_cast<void*>(static_cast<const volatile void*>(_Location)))

```

Comment:
HM in the uninitialized_meow PR we introduced a voidify Funktion. If it is not too far out we should wait until it is merged or reproduce it here

Reply:
The whole point of pushing this through before finishing test coverage is to avoid blocking the compiler team. If we get `uninit_meow` merged in time, I'll make the `voidify` change here. Otherwise, I'll do so in the followup. (Leaving this comment open as a reminder.)

Reply:
:+1:

Reply:
MSVC-PR-276291 is internally out for review, containing `uninit_meow`, so it's a race :rocket:

Reply:
I've annotated the `destroy`/`destroy_n` test coverage followup work item in #39 to mention integration with GH-1164 and `_Voidify_iter` specifically.